src/coreclr/src/tools/Common/TypeSystem/Common/TypeSystemContext.cs

```diff
@@ -66,7 +66,7 @@ public virtual ModuleDesc ResolveAssembly(AssemblyName name, bool throwIfNotFoun
             return null;
         }
 
-        internal virtual ModuleDesc ResolveModule(IAssemblyDesc referencingModule, string fileName, bool throwIfNotFound = true)
+        ModuleDesc IModuleResolver.ResolveModule(IAssemblyDesc referencingModule, string fileName, bool throwIfNotFound)
```

Comment:
ILVerify supports multi-module assemblies and overrides this. I think this will be a build break there. Maybe leaving it as non-explicit implementation is enough?

Reply:
Fixed by putting the internal virtual method back and calling that from IModuleResolver.ResolveModule.